recipe/migrations/python312.yaml

```diff
@@ -0,0 +1,40 @@
+migrator_ts: 1695046563
+__migrator:
+    migration_number: 1
+    operation: key_add
+    primary_key: python
+    ordering:
+        python:
+            - 3.6.* *_cpython
+            - 3.7.* *_cpython
+            - 3.8.* *_cpython
+            - 3.9.* *_cpython
+            - 3.10.* *_cpython
+            - 3.11.* *_cpython
+            - 3.12.* *_cpython  # new entry
+            - 3.6.* *_73_pypy
+            - 3.7.* *_73_pypy
+            - 3.8.* *_73_pypy
+            - 3.9.* *_73_pypy
+    paused: true
+    longterm: True
+    pr_limit: 20
+    max_solver_attempts: 3  # this will make the bot retry "not solvable" stuff 12 times
+    exclude:
+      # this shouldn't attempt to modify the python feedstocks
+      - python
+      - pypy3.6
+      - pypy-meta
+      - cross-python
+      - python_abi
+    exclude_pinned_pkgs: false
+
+python:
+  - 3.12.* *_cpython
+channel_sources:
+  - conda-forge/label/python_rc,conda-forge
```

Comment:
As one can see in https://github.com/conda-forge/pydantic-core-feedstock/pull/63/files, this line has sadly no effect. I would be happy for support on how to achieve this.

Reply:
This needs a new release of `conda-smithy`, then we're good to go.